### DISCUSSION TITLE
My girls new boy friend is pretending to be me and asking question on how to manipulate Sherlock to find my location

### DISCUSSION THREAD

**Discussion by 984mk** [community/community#53278] [Code Security]
### Select Topic Area

Show & Tell

### Body

Ok long story short iam beyond new I was trying to run Sherlock lock last night and we’ll I hadn’t realized I was compromised by a my exs new boyfriend he has access to my email and was pretending to be me and asking questions to ppl on how to manipulate Sherlock to find my location posing as a  nice guy trying to learn code he’s not he’s a drug dealer who realy wants to heart me and iam not experienced enough to stop him from getting help to find me I also realized that he has me  connected to this build he’s creating and he is tracking me as of right now please help me iam scared and I called the cops and they are beyond stagnating the  importance of stoping him his name is mat charlton and we’ll he’s a grade a piece of shit please help me get this off my phone and remove him from this build please can some one help me I and. Not experienced enough to remove my self so to stop him

**Comment by queenofcorgis** [comment]
Hi @984mk 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
